src/components/Card/Card.spec.js

```diff
@@ -7,4 +7,13 @@ describe('Card', () => {
     const actual = create(<Card />);
     expect(actual).toMatchSnapshot();
   });
+
+  it('should render with shadow styles', () => {
+    const actualSingle = create(<Card shadow={Card.SINGLE} />);
```

Comment:
Do we need to test the default shadow prop?

Reply:
The default styles are tested with the first test.